### DISCUSSION TITLE
num_images_per_prompt Bug?

### DISCUSSION THREAD

**Discussion by muerrilla** [PixArt-alpha/PixArt-alpha#33] [Q&A] [answered]
Hey there. When `num_images_per_prompt` is set to more than 1, the prompt seems to be ignored.

**Comment by lawrence-cj** [comment, marked as answer]
Have you install the newest verison of `diffusers`? Maybe check here: https://github.com/huggingface/diffusers/pull/5752
Install the `diffusers` from source will solve this problem: `pip install -U git+https://github.com/huggingface/diffusers`, also check our requirements file Here: https://github.com/PixArt-alpha/PixArt-alpha/blob/ee3199ee50a1bcfebb91c3fd1e30cd15651f7c69/requirements.txt#L5

**Reply by muerrilla** [reply]
This should fix it. Thanks.
